data_prep/introspector.py

```diff
@@ -260,10 +361,11 @@ def _contains_function(funcs: List[Dict], target_func: Dict):
   return False
 
 
-def _postprocess_function(target_func: Dict):
+def _postprocess_function(target_func: dict, project_name: str) -> dict:
   """Post-processes target function."""
-  target_func['return-type'] = _get_clean_return_type(target_func)
+  target_func['return-type'] = _get_clean_return_type(target_func, project_name)
   target_func['function-name'] = demangle(target_func['function-name'])
+  return target_func
```

Comment:
Why do we need to return the dictionary?

Reply:
Thinking it will help avoid ambiguity between pass by reference and value, although dict and list are by reference. Happy to change it back if it is not necessary :)

Reply:
I see.
Let's change it back for simplicity.
Thanks!